src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -220,6 +219,7 @@
     <Compile Include="System\Drawing\SystemIcons.Windows.cs" />
     <Compile Include="System\Drawing\ToolboxBitmapAttribute.cs" />
     <Compile Include="System\Drawing\Text\PrivateFontCollection.Windows.cs" />
+    <Compile Include="System\Drawing\UnsafeNativeMethods.cs" />
```

Comment:
Nit: It would be nice to refactor this to follow the interop coding conventions while you are on it.

Reply:
Sounds good. I'll update to do it.